benchmarks/lucet-benchmarks/src/modules.rs

```diff
@@ -215,33 +215,32 @@ pub fn many_args_mock() -> Arc<dyn Module> {
 }
 
 pub fn hostcalls_mock() -> Arc<dyn Module> {
-    lucet_hostcalls! {
```

Comment:
Note that throughout this PR, whenever you see `lucet_hostcalls!` replaced by `#[lucet_hostcall]`, know that nothing in the body of the function has actually changed, just the syntax required to move from the old macro to the new one.